src/components/NewBeer.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+function NewBeer() {
+
+    const [beersArr, setBeersArr] = useState({
```

Comment:
The name should be descriptive of what you have or what you're trying to have

```suggestion
    const [newBeer, setNewBeer] = useState({
```